frontend/src/reviews/ReviewCommentDisplay.tsx

```diff
@@ -0,0 +1,40 @@
+import { Box, Card, Divider, Stack, Typography } from '@mui/material'
+import UserAvatar from 'src/common/UserAvatar'
+import { EntityKind, ReviewComment } from 'types/types'
+import { formatDateString } from 'utils/dateUtils'
+
+type ReviewCommentsProps = {
+  response: ReviewComment
+}
+
+export default function ReviewComment({ response }: ReviewCommentsProps) {
+  const username = response.user.split(':')[0]
```

Comment:
This is the ID type, not the username!  It'll call all users 'user'.  You probably want `[1]` instead.